src/stores/PostIdsStatuses.ts

```diff
@@ -88,26 +95,31 @@ subscribeKey(PostStore, 'selectedPosts', updateStatusesForSelectedPosts)
 setInterval(() => updateStatusesForSelectedPosts(), 5000)
 
 setInterval(async () => {
-  for (const name in postStatusStore.processing) {
+  for (const name in Object.keys(data)) {
+    if (!postStatusStore.statuses[name]) return
+    const ids: number[] = []
+
+    await Promise.all(
+      Object.entries(postStatusStore.statuses[name]).map(
+        async ([blockchainId, postData]) => {
+          const { status } = await postData
+          if (status === PostStatus.pending || status === PostStatus.approved)
```

Comment:
adds only pending and approved posts into ids that should be checked, because we shouldn't check rejected or published, those statuses can't be changed anymore